commit 0805c333a5d38ad2ff93a763264c6c395957210a
Author: Orlando Vazquez <ovazquez@gmail.com>
Date:   2017-09-13T01:56:07-07:00 (2 years, 1 month ago)
    
    CNAPI-718 CNAPI returns 500 Internal Error for request param validation errors

diff --git a/lib/validation/endpoints.js b/lib/validation/endpoints.js
index fad2296..8bbe59c 100644
--- a/lib/validation/endpoints.js
+++ b/lib/validation/endpoints.js
@@ -227,7 +227,7 @@ function ensureParamsValid(req, res, paramRules, opts) {
     if (errors) {
         errors = formatValidationErrors(errors);
         res.send(
-            500,
+            409,
             errors);
         return errors;
     }
